TASK: Adjust to Flow Controller changes

kitsunet opened the pull request:
See https://github.com/neos/flow-development-collection/pull/3311

<!-- 
    Thanks for your contribution, we appreciate it!

    The first section should explain briefly what is changed. 
    Some examples are always nice to showcase the use. 
    The content will be used in the change-logs and addresses 
    developers working with Neos.

    If there are issues regarding the topic of your PR link 
    them here as `related:` or `resolved:`
-->

**Upgrade instructions**

<!-- 
    Add upgrade instructions for breaking changes. 
    Explain who is affected, what has to be adjusted  
-->

**Review instructions**

<!-- 
    If your change is not explained fully by the first section you can 
    add more details here to help the reviewers understand the change.
    We have to understand what you did, why you did it and how we can 
    verify it works correctly and does no harm.
-->

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

kitsunet committed:
TASK: WIP Adjust to Flow Controller changes

mhsdesign commented on Neos.Neos/Classes/FrontendRouting/SiteDetection/SiteDetectionMiddleware.php:
```diff
@@ -42,6 +42,7 @@ final class SiteDetectionMiddleware implements MiddlewareInterface
 
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
+//        return $handler->handle($siteDetectionResult->storeInRequest($request));
```
räusper räusper

mhsdesign commented on Neos.Neos/Configuration/Routes.Service.yaml:
```diff
@@ -42,6 +42,17 @@
     '@format':     'json'
   httpMethods: ['GET']
 
+- name: 'Services - UserPreferencesController->redirect'
+  uriPattern: 'user-preferences-redir'
+  defaults:
+    '@package': 'Neos.Neos'
+    '@subpackage': 'Service'
+    '@controller': 'UserPreference'
+    '@action': 'redirect'
+    '@format': 'html'
+  httpMethods: [ 'GET' ]
```
what is up with this? Probably just for testing?

kitsunet committed:
Merge branch '9.0' into task/adjust-to-flow-changes

mhsdesign committed:
WIP Revert use of SimpleActionController

mhsdesign committed:
WIP

mhsdesign committed:
Merge remote-tracking branch 'origin/9.0' into task/adjust-to-flow-changes

mhsdesign committed:
TASK: Revert testing adjustments

mhsdesign committed:
TODO REVERT ME !!! Just for ci as hotfix

mhsdesign committed:
WIP: TASK Adjust to PsrResponse from dispatcher

see https://github.com/neos/flow-development-collection/pull/3294

mhsdesign committed:
WIP: Fix plugin unit test

mhsdesign committed:
Merge remote-tracking branch 'origin/9.0' into task/adjust-to-flow-changes

mhsdesign committed:
Merge remote-tracking branch 'origin/9.0' into task/adjust-to-flow-changes

mhsdesign committed:
TASK: Simplify module controller request handling

and incorporate fix https://github.com/neos/neos-development-collection/pull/5179 for Neos 9

mhsdesign committed:
TASK: Revert Pipeline hack

mhsdesign reviewed (approved):
Works locally

mhsdesign committed:
TASK: Fix unit test after flow psr adjustments
